docs/content/tutorials/risky-containers/_index.md

```diff
@@ -0,0 +1,301 @@
+- Press Cmd + Shift + P (on MacOS) or Ctrl + Shift + P (Windows or Linux) to
+    launch the command palette.
+- Select `Dev Containers: Rebuild and Reopen in Container`.
+- Select the `Risky Containers with Drasi` option to launch this tutorial.
+
+{{% /tab %}}
+
+{{% tab header="Local Setup" text=true %}}
+
+You need to have your own Kubernetes cluster setup.
+You can use any Kubernetes setup.
+For a local testing setup, you can choose one of alternatives
+  like Kind, Minikube or k3d.
+
+Make sure that `kubectl` on your system points to your Kubernetes cluster.
+
+You will need [VS Code](https://code.visualstudio.com/)
+
+You will need the [Drasi VS Code extension](https://marketplace.visualstudio.com/items?itemName=DrasiProject.drasi)
+
+You will need the [PostgreSQL CLI tool](https://www.postgresql.org/download/)
+
+You will need to deploy PostgreSQL to your cluster. The following command can be used, it will also create the table and data required for this tutorial.
+
+```shell
+kubectl apply -f ./resources/postgres.yaml 
+kubectl wait --for=condition=ready pod -l app=postgres --timeout=60s
+```
+
+If you are not using the GitHub codespace or VS Code dev container, you will need to open a port forward to access the PostgreSQL instance.
+    
+```shell
+kubectl port-forward services/postgres 5432:5432
+```
+
+{{% /tab %}}
+
+{{% /tabpane %}}
+
+### PostgreSQL Table
+
+A PostgreSQL table named `RiskyImage` has been pre-loaded with that following data:
+
+
+| Id  | Image                             | Reason        |
+|-----|-----------------------------------|---------------|
+| 1   | drasidemo.azurecr.io/my-app:0.1   | Security Risk |
+| 2   | docker.io/library/redis:6.2.3-alpine   | Compliance Issue  |
+
+
+### Deploy Pods
+
+The following command will deploy two Pods of `my-app`, one with version `0.1` and one with version `0.2`. 
+
+```shell
+kubectl apply -f ./resources/my-app.yaml  
+```
+
+
+### Store Kubernetes credentials in a secret
+
+Before we can create a Kubernetes source, we need the credentials of the cluster that the source will connect to. The way to get these credentials will differ depending on how you are running Kubernetes.  The scripts below will extract the credentials of your current Kubernetes context and store them in a secret, to be referenced by the Drasi Kubernetes source.
+
+> If you are running using Github Codespaces or the VS code dev container, then use the `k3d` script.
+
+{{< tabpane >}}
+
+{{% tab header="k3d" text=true %}}
+
+```shell
+k3d kubeconfig get k3s-default | sed 's/0.0.0.0.*/kubernetes.default.svc/g' | kubectl create secret generic k8s-context --from-file=context=/dev/stdin -n drasi-system
+```
+
+{{% /tab %}}
+
+{{% tab header="Kind" text=true %}}
+
+```shell
+kind get kubeconfig | sed 's/127.0.0.1.*/kubernetes.default.svc/g' | kubectl create secret generic k8s-context --from-file=context=/dev/stdin -n drasi-system
+```
+
+{{% /tab %}}
+
+{{% tab header="AKS" text=true %}}
```

Comment:
Should we add EKS? I can test the Source on an EKS cluster and add this part in a future PR